specifyweb/frontend/js_src/lib/utils/cache/definitions.ts

```diff
@@ -80,6 +80,8 @@ export type CacheDefinitions = {
   } & {
     readonly /** Open nodes in a given tree */
     [key in `conformations${AnyTree['tableName']}`]: Conformations;
+  } & {
+    readonly [key in `definition${AnyTree['tableName']}`]: number;
```

Comment:
sorry if I missed it, but I don't see where this cache entry is set - I only see it being read
is the code for setting it not developed yet?
and how is the case when this isn't set yet handled (first start of specify in incognito tab)

Reply:
it's set when the user changes the taxon tree in the tree viewer

Reply:
it will be the first available for now and we will add some new logic and a new way to define the default tree that is being used